htdocs/comm/address.php

```diff
@@ -106,7 +106,7 @@
         }
         else
         {
-	        setEventMessages($object->error, $object->errors, 'errors');
+	        setEventMessages($object->error, null, 'errors');
```

Comment:
If we have $object->error as first param, we should have $object->errors at second. Why removing this ?

Reply:
The property errors does not exist on Address and this class is deprecated